packages/graph/hash_graph/lib/graph/src/store/postgres/query/compile.rs

```diff
@@ -67,15 +68,22 @@ impl<'f: 'q, 'q, T: PostgresQueryRecord<'q>> SelectCompiler<'f, 'q, T> {
     }
 
     /// Adds a new path to the selection.
-    pub fn add_selection_path(&mut self, path: &'q T::Path<'q>) {
+    ///
+    /// Optionally, a path can be ordered by passing an [`Ordering`] alongside the path.
```

Comment:
Is the path ordered? The values at the path? Are we ordering _by_ the values on the path?